setup.py

```diff
@@ -274,7 +274,9 @@ def getpyexts():
     ela = []
 
     if using_intel and IS_WIN:
-        if os.environ.get('cc', '') == "dpcpp":
+        if os.environ.get('cc', '') == "dpcpp" or \
+                os.environ.get('cc', '') == "clang++" or \
+                os.environ.get('cc', '') == "clang-cl":
```

Comment:
Probably, will be easier:
```suggestion
        if os.environ.get('cc', '') in ("dpcpp", "clang++", "clang-cl"):
```

Reply:
Good point, thanks!